app/components/rails_bootstrap/badge_component.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class RailsBootstrap::BadgeComponent < ApplicationComponent
+
+  VARIATIONS = ["primary", "secondary", "success", "danger", "warning", "light", "info", "dark"].freeze
+
+  validates :variation, inclusion: { in: VARIATIONS }
+  validates :pill, inclusion: { in: [true, false] }
+
+  def initialize(variation: "primary", pill: false)
+    @variation = variation
+    @pill = pill
+  end
+
+  def styles
+    "badge" + pill_class + variation_class
+  end
+
+  private
+    
+    def pill_class
+      @pill ? " badge-pill" : ""
```

Comment:
It is always suggested that we never call instance variables directly in Ruby. In fact the reason we have `attr_reader` in this class is so we can hide the instance variable within a private method named `pill`. Can you change this `@pill` to `pill` please